configure.py

```diff
@@ -643,6 +643,7 @@ def find_ninja():
     'test/perf/perf_idl',
     'test/perf/perf_vint',
     'test/perf/perf_big_decimal',
```

Comment:
> sort_by_proximity_topology.perf_sort_by_proximity      994665   996.540ns     0.900ns   995.639ns   998.420ns       0.000       0.000     13230.4      2982.9

13k instructions to sort by proximity?!

Reply:
I'll check if that's normalized by iteration, as each call does 64 iterations.

Reply:
It is normalized by the number of iterations in each run.
See https://github.com/scylladb/seastar/blob/master/tests/perf/perf_tests.cc#L332

Reply:
So we need 1100 instructions to sort three items?

Reply:
At least you get 4.5IPC.

btw std::sort is likely terrible for such low counts. Bubble sort would likely be better.

Reply:
(or insertion sort)